### DISCUSSION TITLE
Allow withTrashed on groups

### DISCUSSION THREAD

**Discussion by Edsardio** [laravel/framework#38733] [Ideas]
If you have a huge collection of routes that need to support the [withTrashed()](https://laravel.com/docs/8.x/routing#implicit-soft-deleted-models) functionality it would be nice to be able to use it on an overarching group or something so you don't have to call it multiple times.

**Comment by SuperDJ** [comment]
Made a PR here: https://github.com/laravel/framework/pull/39059
